xclim/indicators/atmos/_conversion.py

```diff
@@ -248,76 +278,102 @@ def cfcheck(self, **das):
         if kws["method"] == "CAN"
         else "Invalid temperatures (T > 50°F) and winds (V < 3 mph) where masked."
     ),
+    abstract="Wind chill factor is an index that equates to how cold an average person feels. "
+    "It is calculated from the temperature and the wind speed at 10 m. "
+    "As defined by Environment and Climate Change Canada, a second formula is used for light winds. "
+    "The standard formula is otherwise the same as used in the United States.",
     compute=indices.wind_chill_index,
     parameters={"mask_invalid": True},
 )
 
 
 potential_evapotranspiration = Converter(
+    title="Potential evapotranspiration",
     identifier="potential_evapotranspiration",
     var_name="evspsblpot",
     units="kg m-2 s-1",
     standard_name="water_potential_evapotranspiration_flux",
     long_name="Potential evapotranspiration",
     description=(
         "The potential for water evaporation from soil and transpiration by plants if the water "
-        "supply is sufficient, with the method {method}."
+        "supply is sufficient, calculated with the {method} method."
+    ),
+    abstract=(
+        "The potential for water evaporation from soil and transpiration by plants if the water "
+        "supply is sufficient, calculated with a given method."
     ),
     compute=indices.potential_evapotranspiration,
 )
 
 water_budget_from_tas = Converter(
+    title="Water budget",
     identifier="water_budget_from_tas",
     units="kg m-2 s-1",
     long_name="Water budget",
     description=(
         "Precipitation minus potential evapotranspiration as a measure of an approximated surface water budget, "
-        "where the potential evapotranspiration is calculated with the method {method}."
+        "where the potential evapotranspiration, calculated with the {method} method."
```

Comment:
```suggestion
        "where the potential evapotranspiration is calculated with the {method} method."
```
I think in this case the method refers only to the potential evap calculation so the old version is what we want